app/controllers/cinema_halls_controller.rb

```diff
@@ -0,0 +1,50 @@
+class CinemaHallsController < ApplicationController
```

Comment:
Just a small thing that caught my attention. Naming is tricky sometimes - we usually want to keep it as simple and straight forward as much as possible. And in context of whole project "Monte Cinema" a Hall is specific enough and does not need additional prefix as "Cinema".
What do you think? :) It's up to you of course

Reply:
I thought about it and it's difficult to find a good reason to stick with the CinemaHall name, decided to rename it to Hall as you suggested.